package.json

```diff
@@ -16,7 +16,10 @@
     "format": "turbo format --continue -- --cache --cache-location node_modules/.cache/.prettiercache --ignore-path ./.prettierignore",
     "format:fix": "turbo format --continue -- --write --cache --cache-location node_modules/.cache/.prettiercache  --ignore-path ./.prettierignore",
     "lint": "turbo lint --continue -- --cache --cache-location node_modules/.cache/.eslintcache",
-    "lint:fix": "turbo lint --continue -- --fix --cache --cache-location node_modules/.cache/.eslintcache"
+    "lint:fix": "turbo lint --continue -- --fix --cache --cache-location node_modules/.cache/.eslintcache",
+    "lint:ws": "npx sherif@latest",
+    "lint:ws:fix": "npx sherif@latest --fix",
```

Comment:
being annoying here but the [package itself](https://github.com/QuiiBz/sherif?tab=readme-ov-file#installation) suggests pinning to a version rather than `latest`. not entirely sure what the example meant but their principle of preventing regressions was useful i thought.

> This is useful to prevent regressions (e.g. when adding a library to a package but forgetting to update the version in other packages of the monorepo).